dashboard/config/routes.rb

```diff
@@ -7,6 +7,9 @@
   # Redirect studio.code.org/courses to code.org/students
   get "/courses", to: redirect(CDO.code_org_url("/students"))
 
+  # Redirect studio.code.org/mysection to most recent section section
+  get '/teacher_dashboard/sections/firstSectionProgress', to: "teacher_dashboard#redirect_to_newest_section"
```

Comment:
```suggestion
  get '/teacher_dashboard/sections/first_section_progress', to: "teacher_dashboard#redirect_to_newest_section"
```
Looks like these should be underscore_case. Also, It might make sense to run the name by Bryan if you haven't already

Reply:
@dju90 thoughts on the formatting of the URL or the name?  This would be easy to change also if you had an entirely different vision :)

Reply:
Since the URL/name redirects immediately, I don't have a strong opinion what it should be. `first_section_progress` sounds good, as does `newest_section_progress` if you're looking for an alternative

Reply:
Great - let's go with `first_section_progress`